src/services/item/plugins/itemTag/repository.ts

```diff
@@ -60,14 +60,22 @@ export const ItemTagRepository = AppDataSource.getRepository(ItemTag).extend({
       }),
     );
 
-    const hasTags = await query
+    const hasTags: ItemTag[] = await query
       .andWhere('itemTag.type IN (:...types)', { types: tagTypes })
-      .getOne();
+      .getMany();
```

Comment:
Info: why did the previous implementation not return many?

Reply:
The previous implementation didn't work and wasn't used